### PR TITLE
added documentation for image driver forcing file

### PR DESCRIPTION
**Pull request by dgergel**
This PR addresses #548.

### CODE REVIEW

File: docs/Documentation/Drivers/Image/ForcingData.md [hunk 1]
```diff
@@ -1,2 +1,103 @@
...
+| PRESSURE   | Atmospheric pressure                | kPa             |   
+| SWDOWN     | Incoming shortwave radiation        | W/m<sup>2</sup> |
+| LWDOWN     | Incoming longwave radiation         | W/m<sup>2</sup> |
+| VP         | Vapor pressure                      | kPa             |   
+| WIND       | Wind speed                          | m/s             |   
+
+Example output from `ncdump -h forcingfile.nc` should look like this: 
+
+```
+netcdf Stehekin_image_test.forcings_10days.1949 {
```
Comment (jhamman) [inline docs/Documentation/Drivers/Image/ForcingData.md]: `ncdump -h forcingfile.nc` --> `ncdump -h Stehekin_image_test.forcings_10days.1949`

which reminds me, you should mention that the files must be provided in chunks of individual years (e.g. `prefix.$year.nc`).

File: docs/Documentation/Drivers/Image/ForcingData.md [hunk 2]
```diff
@@ -1,2 +1,103 @@
-!!! Warning
-    Docs not written yet.
+# VIC Forcing File
+
+The VIC Image Driver requires a NetCDF file with gridded subdaily forcings.  The required forcing variables and units are listed below (and must also be specified in the global parameter file):
```
Comment (jhamman) [inline docs/Documentation/Drivers/Image/ForcingData.md]: link to section of global parameter file.

File: docs/Documentation/Drivers/Image/ForcingData.md [hunk 3]
```diff
@@ -1,2 +1,103 @@
...
+        :conventions = "CF-1.6" ;
+        :hostname = "compute-0-2.local" ;
+        :username = "ymao" ;
+        :version = "VIC.4.2.b" ;
+        :grid = "BPA Grid" ;
+}
+
+```
+
+Using gridded subdaily forcings for vegetation, lake and carbon cycle timeseries is under development.
```
Comment (jhamman) [inline docs/Documentation/Drivers/Image/ForcingData.md]: ...The documentation for these features will be added at a later date.

**Comment by jhamman** [comment]
looks good. just a few minor suggestions.

**Comment by dgergel** [comment]
@jhamman this should be ready to merge.

File: docs/Documentation/Drivers/Image/ForcingData.md (hunk 1 above)
Comment (jhamman) [inline docs/Documentation/Drivers/Image/ForcingData.md]: One fix remaining.

Comment (dgergel) [inline docs/Documentation/Drivers/Image/ForcingData.md]: Updated. Sorry that I missed this earlier.

**Pull request merged**